mage_ai/frontend/components/DataTable/index.tsx

```diff
@@ -249,23 +253,37 @@ function Table({
             >
               {firstColumn && cell.render('Cell')}
               {!firstColumn && (
-                <>
-                  {cellValue === true && 'true'}
-                  {cellValue === false && 'false'}
-                  {(cellValue === null || cellValue === 'null') && 'null'}
-                  {cellValue !== true
+                <FlexContainer justifyContent={'space-between'}>
+                  <Text danger={isInvalid} wordBreak>
+                    {cellValue === true && 'true'}
+                    {cellValue === false && 'false'}
+                    {(cellValue === null || cellValue === 'null') && 'null'}
+                    {cellValue !== true
                     && cellValue !== false
                     && cellValue !== null
                     && cellValue !== 'null'
                     && cellValue
                   }
-                </>
+                  </Text>
+                  {isInvalid && (
+                    <NextLink
+                      as={`/datasets/${slug}/?tabs[]=Reports&show_columns=1&column=${index}`}
```

Comment:
There are `TABS_QUERY_PARAM` and `SHOW_COLUMNS_QUERY_PARAM` constants from the `mage-ai/mage_ai/frontend/components/datasets/overview/index.tsx` file you can use here. No `COLUMN_QUERY_PARAM` but you can add it. May be helpful if we end up changing the query params down the road, will only need to change in one place.